RPG Wizardry/Assets/Scripts/GameWorld/FloorManager.cs

```diff
@@ -24,7 +24,8 @@ public class FloorManager : SingletonBehaviour<FloorManager>
         /// <summary>
         /// Currently Loaded Room
         /// </summary>
-        private Room activeRoom;
+        [HideInInspector]
+        public Room activeRoom;

```

Comment:
Make the setter private, as Rooms should only be loaded via the Methods. (You then also shouldn't need the [HideInInspector] anymore)

Reply:
Make the Casing fit the accessibility as well (UpperCamelCase)